dsnp-wallet/dsnp-wallet/Source/ViewControllers/TabBarViewController/TabBarViewController.swift

```diff
@@ -21,7 +21,8 @@ class TabBarViewController: UITabBarController {
         let appearance = UITabBarAppearance()
         appearance.configureWithOpaqueBackground()
         appearance.backgroundColor = .white
-        appearance.stackedLayoutAppearance.selected.iconColor = .black
+        appearance.stackedLayoutAppearance.selected.iconColor = UIColor.Theme.accentOrange
+        appearance.stackedLayoutAppearance.selected.titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.black]
         self.tabBar.standardAppearance = appearance
         if #available(iOS 15.0, *) {
             self.tabBar.scrollEdgeAppearance = self.tabBar.standardAppearance
```

Comment:
It's below this line, but you have a block where you're customizing the back bin config but we don't currently have a back btn. Let's remove for now.

Reply:
Also, we have UIImage.Theme.backArrow pointing to an asset that does not exist in the project. Let's remove backArray from UIImage+Theme.